data/text/text_5.asm

```diff
@@ -36,7 +36,7 @@ _FlashLightsAreaText::
 	prompt
 
 _WarpToLastPokemonCenterText::
-	text "Warp to the last"
+	text "warp_event to the last"
```

Comment:
This should definitely stay as-is; it's at least one reason why `make compare` is failing.